src/components/SwapWidget/subcomponents/InfoSection/InfoSection.tsx

```diff
@@ -116,6 +125,88 @@ const InfoSection: FC<InfoSectionProps> = ({
     );
   }
 
+  if (isPairUnavailable) {
```

Comment:
There's still duplicate code in here.

Reply:
I removed duplicate code. I ran through test transactions on Goerli and all text seems to render correctly

Reply:
Again, still too many changes. The only change should be:

```
if (hasSelectedCustomServer) {
    return (
      <>
        <StyledInfoHeading>
          {t("orders.selectedServer", { serverUrl })}
        </StyledInfoHeading>
        <ClearServerButton handleClearServerUrl={handleClearServerUrl} />
      </>
    );
  }
```

Reply:
I copy pasted in code from `develop` and added in this code snippet to line 94